PendingReleaseNotes.md

```diff
@@ -18,3 +18,4 @@ v1.5...
 
 * Ceph Block Pool: add mirroring support
 * Ceph Block Pool: add `replicasPerFailureDomain` to set the number of replica in a failure domain ([#5591](https://github.com/rook/rook/issues/5591))
+* ceph: export the storage capacity of the ceph cluster ([#6475](https://github.com/rook/rook/pull/6475))
```

Comment:
```suggestion
* Ceph Cluster: export the storage capacity of the cluster ([#6475](https://github.com/rook/rook/pull/6475))
```

We tend to prefix with the CRD name that changed.

Reply:
Done.